README.md

```diff
@@ -1,2 +1,12 @@
 # Argos
-Static code analyzer.
+
+## Development
+Argos uses java 17, its package manager is Gradle. The project has been dockerized.
+
+### Recommended commands
+Run `$ docker compose build` to *create a new docker image* with the latest changes.
+Run `$ docker compose up -d` to *start* the project in the background.
+Run `$ docker logs <service-name>` to access the logs of a service.
+
+For more information, review the documentation about [docker compose](https://docs.docker.com/reference/cli/docker/compose/)
+
```

Comment:
Can you also add about the executable that is in this folder location `build/libs/argos-0.0.1-SNAPSHOT.jar` and also about the swagger?

```
http://localhost:8080/swagger-ui/index.html
```

Also the command to run the test

Reply:
Yes, I'll add the URL of swagger. But what do I do with the JAR?

Reply:
Just take into account what was put in the pr which are these 2 commands for consistency, which was put this:

```
gradle bootjar
```

run the .class with the following command:

```
./gradlew bootRun
```

To run the tests:
```
./gradlew test
```

The first command you enter is only so that the developers know how to create the jar

Reply:
Oh, yes indeed. I could add indications on how to run the tests. But I think we should have only one official way of running the project, let's pick docker as official.

Reply:
I think it's good